frontend/src/network/notes_api.ts

```diff
@@ -1,11 +1,18 @@
 import { Note } from "../models/note";
+import { DeletedNote } from "../models/deletedNote";
 import { fetchData } from "./fetch_data";
 
 export async function fetchNotes(): Promise<Note[]> {
   const response = await fetchData("/api/notes", {
     method: "GET",
   });
-  return response.json();
+
+  const notes = await response.json();
+  const filteredNotes = notes.filter(
```

Comment:
Please be descriptive at your naming, yes these are filtered notes but they are also `existing` notes or `live` notes as the deletedAt is undefined, you can use one of these adjectives on naming this variable.